ui-sdk-components-sample/app/src/main/res/layout/fragment_trip_quotes.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/sky_blue"
+    android:padding="@dimen/padding_default"
+    tools:context=".quotes.TripQuotesFragment">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/quotes_layout"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:background="@color/light_blue"
+        android:padding="@dimen/padding_default"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <androidx.core.widget.ContentLoadingProgressBar
+            android:id="@+id/loadingProgressBar"
+            style="?android:attr/progressBarStyle"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:visibility="invisible"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintTop_toTopOf="parent"/>
+
+        <TextView
+            android:id="@+id/header_label"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:text="@string/quotes_header"
+            android:textAppearance="@style/TextAppearance.AppCompat.Title"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"/>
+
+        <com.karhoo.uisdk.screen.booking.supplier.SupplierListView
```

Comment:
Can you move the SupplierListView to the bottom of the screen: 

Could try the below 
```
<?xml version="1.0" encoding="utf-8"?>
<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:background="@color/sky_blue"
    android:padding="@dimen/padding_default"
    tools:context=".quotes.TripQuotesFragment">

    <androidx.constraintlayout.widget.ConstraintLayout
        android:id="@+id/quotes_layout"
        android:layout_width="0dp"
        android:layout_height="wrap_content"
        android:background="@color/light_blue"
        android:padding="@dimen/padding_default"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintTop_toTopOf="parent">

        <androidx.core.widget.ContentLoadingProgressBar
            android:id="@+id/loadingProgressBar"
            style="?android:attr/progressBarStyle"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:visibility="invisible"
            app:layout_constraintStart_toStartOf="parent"
            app:layout_constraintEnd_toEndOf="parent"
            app:layout_constraintBottom_toBottomOf="parent"
            app:layout_constraintTop_toTopOf="parent"/>

        <TextView
            android:id="@+id/header_label"
            android:layout_width="0dp"
            android:layout_height="wrap_content"
            android:text="@string/quotes_header"
            android:textAppearance="@style/TextAppearance.AppCompat.Title"
            app:layout_constraintEnd_toEndOf="parent"
            app:layout_constraintStart_toStartOf="parent"
            app:layout_constraintTop_toTopOf="parent"/>


        <Button
            android:id="@+id/get_quotes_button"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_marginTop="@dimen/margin_default"
            android:background="@color/dark_blue"
            android:padding="@dimen/padding_text"
            android:text="@string/retrieve_quotes"
            android:textColor="@color/colorWhite"
            app:layout_constraintStart_toStartOf="parent"
            app:layout_constraintTop_toBottomOf="@id/header_label" />

        <Button
            android:id="@+id/stop_button"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_marginTop="@dimen/margin_default"
            android:padding="@dimen/padding_text"
            android:background="@color/dark_blue"
            android:text="@string/stop"
            android:textColor="@color/colorWhite"
            app:layout_constraintEnd_toEndOf="parent"
            app:layout_constraintTop_toBottomOf="@id/header_label" />


    </androidx.constraintlayout.widget.ConstraintLayout>
   <com.karhoo.uisdk.screen.booking.supplier.SupplierListView
        android:id="@+id/supplier_list_widget"
        android:layout_width="0dp"
        android:layout_height="@dimen/collapsible_pane_expanded_height"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toBottomOf="@id/quotes_layout"/>

</androidx.constraintlayout.widget.ConstraintLayout>
```

Reply:
Just out of interest, why?

Reply:
I've deleted everything and just kept the supplier list component and pinned it to the bottom. Have also done the same with the payment component.